### ISSUE TITLE
Hybrid content manager by default.

### ISSUE THREAD

**Issue by Carreau** [jupyter/notebook#985]
I would propose for (5.0, maybe earlier) 
to bake into the notebook a notion of Hybrid content manager and "mount point" or the dashboard.

The would allow to have content manager that expose in some folder "normals" ipynbs, in others `ipymd`...etc. 

By default, "normal" content manager would be mounted on `/`.

I want that to have **some** notebook as `ipymd` for collaboration.

**Comment by takluyver** [comment]
I think the 'mount points' abstraction works well for something like the Google Drive integration, where a contents manager is exposing an entirely separate storage space. I think we should think some more whether there's a better way to integrate things like ipymd, which is a different format stored in the same space. Having parallel versions of the same directories in order to open different kinds of files seems awkward.

**Comment by minrk** [comment]
Yeah, I wouldn't think this would be the way to support something like ipymd.
